### PR TITLE
Add a render function for keyboard shortcuts panel

### PR DESCRIPTION
**Pull request by caseklim**
## Summary

Add a prop `renderKeyboardShortcutsPanel` that allows a custom panel for keyboard shortcuts to be rendered.

### Default
image

### renderKeyboardShortcutsPanel
image

### CODE REVIEW

File: examples/DayPickerRangeControllerWrapper.jsx [hunk 1]
```diff
@@ -73,7 +74,7 @@ const defaultProps = {
   renderDayContents: null,
   minimumNights: 1,
   isDayBlocked: () => false,
-  isOutsideRange: day => !isInclusivelyAfterDay(day, moment()),
+  isOutsideRange: (day) => !isInclusivelyAfterDay(day, moment()),
```
Comment (ljharb) [inline examples/DayPickerRangeControllerWrapper.jsx]:
```suggestion
  isOutsideRange: day => !isInclusivelyAfterDay(day, moment()),
```

File: examples/DayPickerRangeControllerWrapper.jsx [hunk 2]
```diff
@@ -143,12 +145,13 @@ class DayPickerRangeControllerWrapper extends React.Component {
 
     return (
       <div style={{ height: '100%' }}>
-        {showInputs &&
+        {showInputs
+          && (
```
Comment (ljharb) [inline examples/DayPickerRangeControllerWrapper.jsx]: personally i'd usually do this like `{showInputs && (`

Comment (caseklim) [reply examples/DayPickerRangeControllerWrapper.jsx]: @ljharb fixed and added more tests :)

**Comment by caseklim** [comment]
@ljharb how long should the coverage check take to run? 🤔

**Comment by ljharb** [comment]
This seems to be the issue @noratarano and i were working on; i believe recent permissions changes caused this to be broken. I’m not sure how to fix it :-/

A short term workaround could be to disable coveralls as a required check; a longer term workaround could be to replace coveralls with another service; ideally we can just figure out how to fix coveralls.

**Comment by caseklim** [comment]
@ljharb ahh that'd do it, any chance it's fixed now? coveralls finished running at least but now I'm seeing "Coverage decreased (-68.7%) to 11.246%" on this PR which seems odd

**Comment by ljharb** [comment]
@caseklim yep, should be all set now! try a rebase, and if it's still claiming coverage has dropped then more tests need to be added.

**Comment by caseklim** [comment]
@ljharb but a coverage drop of -68.7% to 11.246%? I added a test for my new render function, and it's complaining about files I didn't even touch. e.g.,

image

**Comment by ljharb** [comment]
It might be related to the issues coveralls was having - ie, if master was updated while coveralls was down, it might not know about the new coverage.

@noratarano can merge through the failing coverage check, which i think is fine here.

**Comment by noratarano** [comment]
@caseklim @ljharb Merging!

**Pull request merged**